cyphernodeconf_docker/templates/gatekeeper/api.properties

```diff
@@ -33,6 +33,7 @@ action_ln_getinfo=watcher
 action_ln_create_invoice=watcher
 action_ln_getconnectionstring=watcher
 action_ln_decodebolt11=watcher
+action_ln_listpeers=watcher
```

Comment:
Also need to add `action_ln_listfunds`, `action_ln_listpays`, `action_ln_getroute` and `action_ln_withdraw` in this file.  You added them in the sample file only...

Reply:
Yep. Will add it to next commit. Thanks.